plugins/CoreHome/templates/_dataTableCell.twig

```diff
@@ -47,7 +47,7 @@
         {% if row.getMetadata('html_label_prefix') %}<span class='label-prefix'>{{ row.getMetadata('html_label_prefix') | raw }}&nbsp;</span>{% endif -%}
         {%- if row.getMetadata('html_label_suffix') %}<span class='label-suffix'>{{ row.getMetadata('html_label_suffix') | raw }}</span>{% endif -%}
 {% endif %}<span class="value">
-    {%- if row.getColumn(column) %}{% if column=='label' %}{{- row.getColumn(column)|rawSafeDecoded -}}{% else %}{{- row.getColumn(column)|number(2,0)|raw -}}{% endif %}
+    {%- if row.getColumn(column) or (column=='label' and row.getColumn(column) == "0") %}{% if column=='label' %}{{- row.getColumn(column)|rawSafeDecoded -}}{% else %}{{- row.getColumn(column)|number(2,0)|raw -}}{% endif %}
```

Comment:
I wonder if maybe `sameas('0')` works maybe? 

Otherwise isn't it basically the same as `row.getColumn(column) or column=='label'` then? Also within this if there is already a `{% if column=='label' %}{{- row.getColumn(column)|rawSafeDecoded -}}` maybe it could be splitted into two `if` statements like 
`if column == label .... html. elseif row.getColumn ...html`